### PR TITLE
KUZ-682: rename DSL keywords

### PR DESCRIPTION
**Pull request by scottinet**
:warning: Breaking change :warning: 

To prevent confusion with Elasticsearch's DSL, we have to rename the following keywords from our real-time DSL:

- "term" => "equals"
- "terms" => "in"

### CODE REVIEW

**Comment by scottinet** [comment]
Oops, I got carried away when replacing term/terms with equals/in. I also replaced them in ES searches in the security modules. :fearful:

**Comment by scottinet** [comment]
Revert changes in security modules. This PR can be reviewed now.

File: features/step_definitions/readDocument.js [hunk 1]
```diff
@@ -102,8 +102,7 @@ var apiSteps = function () {
   });
 
   this.Then(/^I ?(don't)* find a document with "([^"]*)"(?: in field "([^"]*)")?(?: in index "([^"]*)")?$/, function (dont, value, field, index) {
-    var filters = {filter: { term: {}}};
-    filters.filter.term[field] = value;
+    var filters = {filter: { term: { [field]: value }}};
```
Comment (ballinette) [inline features/step_definitions/readDocument.js]: shouldn't it be `{ equals: { [field]: value }` instead of "term" ?

Comment (scottinet) [inline features/step_definitions/readDocument.js]: This test read documents using Elasticsearch, while we replaced "term" with "equals" only for our real-time engine.

**Pull request merged**